azerothcore/azerothcore-wotlk
# fix(DB): Add additional Incendia Agave spawns

## Changes Proposed:
-  Added 4 more Incendia Agave spawns according to the sources.


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5152
- Closes https://github.com/chromiecraft/chromiecraft/issues/344



## SOURCE:

https://www.youtube.com/watch?v=bE1RIbdOaYE
https://www.youtube.com/watch?v=D62TUG1-mq8


## Tests Performed:

- Tested ingame if the objects are re-spawning on 180 sec.
- Tested if the correct item is received.
- Tested if the spawn positions aren't clipping with the ground.
- Tested on docker/linux.


## How to Test the Changes:

1. Go to each of these locations in thousand needles.
  - `.go xyz -4947.665527 -2019.104126 -52.501392`
  - `.go xyz -5031.855469 -2040.091675 -51.542252`
  - `.go xyz -5027.564941 -2005.562378 -53.395096`
  - `.go xyz -4993.604004 -1971.745728 -53.395203`
2. Loot the node while having quest(5062), you should get the quest item.
2. Loot the node while not having quest(5062), you shouldn't get the quest item.
3. Check if the respawn time is 180 seconds.
4. Check if the node is possitioned well and if its not in some way cliping with the ground.

## Known Issues and TODO List:



## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
